lib/handler.js

```diff
@@ -60,10 +60,22 @@ module.exports = function(options) {
                 _index: options.index
                   || `${indexPrefix}${utils.assembleField(parsedRecord, options.indexField, separator)}`,
                 _type: options.type
-                  || utils.assembleField(parsedRecord, options.typeField, separator),
+                  || (
+                    options.typeField
+                    && utils.assembleField(parsedRecord, options.typeField, separator)
+                  ),
                 _id: id
               };
 
+              // Omit blank _type when ES >= 6
```

Comment:
The new ES client library does not support `apiVersion` attribute. Since there is a plan to upgrade this library to use it, should we just allow the type to be not present without checking the version? Thoughts?

Reply:
I think that would be cleaner. I was trying not to break backward compatibility, but if you're willing to make a minor or major release then I see no problems and it would be more future-proof.

Reply:
@AntonBazhal FYI the reason I did it this way is because of what you said here: https://github.com/AntonBazhal/dynamo2es-lambda/issues/41#issuecomment-388831433

If you decide you still want the type config to be validated, just delete the last commit on this branch. Or, perhaps you could do both major and minor releases if you want to support older clients.